diff --git a/Nu/Nu.Template/Program.fs b/Nu/Nu.Template/Program.fs
index ee3f8868e..b8c600500 100644
--- a/Nu/Nu.Template/Program.fs
+++ b/Nu/Nu.Template/Program.fs
@@ -14,7 +14,7 @@ module Program =
         // this specifies the configuration of the game engine's use of SDL
         let sdlConfig = { SdlConfig.defaultConfig with ViewConfig = NewWindow sdlWindowConfig }
 
-        // use the default world config with the above SDL config
+        // this specifies the world config using the above SDL config
         let worldConfig = { WorldConfig.defaultConfig with SdlConfig = sdlConfig }
 
         // point current working directory at program's base directory
diff --git a/Projects/BlazeVector/Program.fs b/Projects/BlazeVector/Program.fs
index 70de06509..d8999ea6d 100644
--- a/Projects/BlazeVector/Program.fs
+++ b/Projects/BlazeVector/Program.fs
@@ -14,7 +14,7 @@ module Program =
         // this specifies the configuration of the game engine's use of SDL
         let sdlConfig = { SdlConfig.defaultConfig with ViewConfig = NewWindow sdlWindowConfig }
 
-        // use the default world config with the above SDL config
+        // this specifies the world config using the above SDL config
         let worldConfig = { WorldConfig.defaultConfig with SdlConfig = sdlConfig }
 
         // point current working directory at program's base directory
diff --git a/Projects/Elmario/Program.fs b/Projects/Elmario/Program.fs
index 0e206a27b..81bff7371 100644
--- a/Projects/Elmario/Program.fs
+++ b/Projects/Elmario/Program.fs
@@ -13,7 +13,7 @@ module Program =
         // this specifies the configuration of the game engine's use of SDL
         let sdlConfig = { SdlConfig.defaultConfig with ViewConfig = NewWindow sdlWindowConfig }
 
-        // use the default world config with the above SDL config
+        // this specifies the world config using the above SDL config
         let worldConfig = { WorldConfig.defaultConfig with SdlConfig = sdlConfig }
 
         // point current working directory at program's base directory
diff --git a/Projects/Nelmish/Nelmish.fs b/Projects/Nelmish/Nelmish.fs
index c4d1a4379..0db6dfa98 100644
--- a/Projects/Nelmish/Nelmish.fs
+++ b/Projects/Nelmish/Nelmish.fs
@@ -26,7 +26,7 @@ type NelmishDispatcher () =
         | Reset -> just 0
 
     // here we describe the content of the game including its one screen, one group, three
-    // button entities, and one text control.
+    // button entities, and one text control
     override this.Content (model, _) =
         [Content.screen "Screen" Vanilla []
             [Content.group "Group" []
diff --git a/Projects/Nelmish/Program.fs b/Projects/Nelmish/Program.fs
index f5e515202..f2622f1e6 100644
--- a/Projects/Nelmish/Program.fs
+++ b/Projects/Nelmish/Program.fs
@@ -13,7 +13,7 @@ module Program =
         // this specifies the configuration of the game engine's use of SDL
         let sdlConfig = { SdlConfig.defaultConfig with ViewConfig = NewWindow sdlWindowConfig }
 
-        // use the default world config with the above SDL config
+        // this specifies the world config using the above SDL config
         let worldConfig = { WorldConfig.defaultConfig with SdlConfig = sdlConfig }
 
         // point current working directory at program's base directory
diff --git a/Projects/Tactics/Program.fs b/Projects/Tactics/Program.fs
index ff7898075..a4890b802 100644
--- a/Projects/Tactics/Program.fs
+++ b/Projects/Tactics/Program.fs
@@ -16,7 +16,7 @@ module Program =
         // this specifies the configuration of the game engine's use of SDL
         let sdlConfig = { SdlConfig.defaultConfig with ViewConfig = NewWindow sdlWindowConfig }
 
-        // use the default world config with the above SDL config
+        // this specifies the world config using the above SDL config
         let worldConfig = { WorldConfig.defaultConfig with SdlConfig = sdlConfig }
 
         // point current working directory at program's base directory
diff --git a/Projects/UserProject/Program.fs b/Projects/UserProject/Program.fs
index ee3f8868e..b8c600500 100644
--- a/Projects/UserProject/Program.fs
+++ b/Projects/UserProject/Program.fs
@@ -14,7 +14,7 @@ module Program =
         // this specifies the configuration of the game engine's use of SDL
         let sdlConfig = { SdlConfig.defaultConfig with ViewConfig = NewWindow sdlWindowConfig }
 
-        // use the default world config with the above SDL config
+        // this specifies the world config using the above SDL config
         let worldConfig = { WorldConfig.defaultConfig with SdlConfig = sdlConfig }
 
         // point current working directory at program's base directory
